Controller/ContentTypeController.php

```diff
@@ -217,7 +232,7 @@ public function viewContentTypeAction($contentTypeId, $languageCode = null)
         $deleteForm = $this->createForm(new ContentTypeDeleteType(), ['contentTypeId' => $contentTypeId]);
 
         return $this->render('eZPlatformUIBundle:ContentType:view_content_type.html.twig', [
-            'language_code' => $languageCode,
+            'language_code' => $this->getVerifiedContentTypeLanguageCode($contentType, $languageCode),
```

Comment:
OK, i.e. ignore prioritisedLanguages, use $languageCode if valid, otherwise use main CT language and notify about it. Should be fine.